project/Assets/Player/PlayerMovement.cs

```diff
@@ -22,7 +28,19 @@ void Update()
             {
                 bird.WakeUp();
             }
-            bird.AddForce(new Vector2(0, flightForce * bird.mass), ForceMode2D.Impulse);
+            Fly();
         }
+
+        LimitSpeed(bird.velocity);
+    }
+
+    void Fly()
```

Comment:
I know there is no difference between not having  access modifier or having private . But i suggest to add the private modifier since it's more readable for everyone specially for someone else working in the same project as you .